app/scripts/controllers/import.js

```diff
@@ -0,0 +1,366 @@
+'use strict';
```

Comment:
- [ ] This may need an equivalant model, changes to app architecture and naming is required. (Existing import model should be named "compatibility" or "migrations")

Reply:
Do this later...